Documentation/PageTsconfig/TceForm.rst

```diff
@@ -63,1113 +65,1199 @@ dots ('.'), these must be escaped with backslash.
+    Expand pidList by this number of levels. Only has an effect, if pidList has a value.
 
-:aspect:`Description`
-   Limit the search to certain pages (and their subpages). When pidList is empty all pages will be included
-   in the search as long as the backend user is allowed to see them.
+..  _tceform-suggest-pidDepth-example:
 
-:aspect:`Example`
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/page.tsconfig
+Example: Set search depth for suggest field
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-      TCEFORM.suggest.default {
-         # sets the pidList for a suggest fields in all tables
-         pidList = 1,2,3,45
-      }
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/page.tsconfig
 
+    TCEFORM.suggest.default {
+        pidList = 6,7
+        pidDepth = 4
+    }
 
-.. index::
-   Suggest wizard; receiver php class
+..  index::
+    Suggest wizard; pid list
+..  _tceform-suggest-pidList:
 
-receiverClass
-~~~~~~~~~~~~~
+suggest.pidList
+~~~~~~~~~~~~~~~
 
-:aspect:`Datatype`
-   PHP class name
+..  confval:: suggest.pidList
+    :name: tceform-suggest-pidList
+    :type: list of values
 
-:aspect:`Description`
-   PHP class alternative receiver class - the file that holds the class should be derived
-   from :code:`\TYPO3\CMS\Backend\Form\Element\SuggestDefaultReceiver`.
+    Limit the search to certain pages (and their subpages). When pidList is empty all pages will be included
+    in the search as long as the backend user is allowed to see them.
 
-:aspect:`Default`
-   :php:`\TYPO3\CMS\Backend\Form\Element\SuggestDefaultReceiver`
+..  _tceform-suggest-pidList-example:
 
+Example: Limit suggest search to records on certain pages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. index::
-   Suggest wizard; rendering user function
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/page.tsconfig
 
-renderFunc
-~~~~~~~~~~
+    TCEFORM.suggest.default {
+        # sets the pidList for a suggest fields in all tables
+        pidList = 1,2,3,45
+    }
 
-:aspect:`Datatype`
-   string
 
-:aspect:`Description`
-   User function to manipulate the displayed records in the result.
+..  index::
+    Suggest wizard; receiver php class
+..  _tceform-suggest.receiverClass:
 
+suggest.receiverClass
+~~~~~~~~~~~~~~~~~~~~~
 
-.. index::
-   Suggest wizard; Where statement
+..  confval:: suggest.receiverClass
+    :name: tceform-suggest-receiverClass
+    :type: Fully Qualified PHP class name
+    :Default: :php:`\TYPO3\CMS\Backend\Form\Element\SuggestDefaultReceiver`
 
-searchCondition
-~~~~~~~~~~~~~~~
+    PHP class alternative receiver class - the file that holds the class should be derived
+    from :code:`\TYPO3\CMS\Backend\Form\Element\SuggestDefaultReceiver`.
 
-:aspect:`Datatype`
-   string
+..  index::
+    Suggest wizard; rendering user function
+..  _tceform-suggest-renderFunc:
+
+suggest.renderFunc
+~~~~~~~~~~~~~~~~~~
+
+..  confval::
```

Comment:
```suggestion
..  confval:: suggest.renderFunc
```

Shoudn't an empty confval throw a warning on rendering?

Reply:
I think that would be better, could you make a PR in the guides?

Reply:
Haven't tested it, was a couch review. That's why the question ;-) So it haven't thrown a warning when rendering on your machine?

Reply:
Done: https://github.com/phpDocumentor/guides/issues/983